pkg/printers/json.go

```diff
@@ -29,6 +30,7 @@ import (
 
 // JSONPrinter is an implementation of ResourcePrinter which outputs an object as JSON.
 type JSONPrinter struct {
+	PathChecker *IllegalPackageSourceChecker
```

Comment:
don't make this configurable, just do it.

Reply:
Not sure what you mean by this, should I be instantiating the checker inside the PrintObj method itself?